src/base-config/keyboard.json

```diff
@@ -293,7 +296,7 @@
             "platform": "mac"
         }
     ],
-    "navigate.nextDoc":  [
+    "next.recent.files":  [
```

Comment:
I do think `recent.files.next` (and the same for below) fits better with our mostly-hierarchical command naming.